developer/keymanweb/index.php

```diff
@@ -42,11 +32,11 @@
 &lt;script src='<?=$cdnUrlBase?>/keymanweb.js'&gt;&lt;/script&gt;
 &lt;script src='<?=$cdnUrlBase?>/kmwuitoggle.js'&gt;&lt;/script&gt;
 &lt;script&gt;
-  (function(kmw) {
-    kmw.init({attachType:'auto'});
-    kmw.addKeyboards('@en'); // Loads default English keyboard from Keyman Cloud (CDN)
-    kmw.addKeyboards('@th'); // Loads default Thai keyboard from Keyman Cloud (CDN)
-  })(keyman);
+  (function() {
+    keyman.init({attachType:'auto'});
+    keyman.addKeyboards('@en'); // Loads default English keyboard from Keyman Cloud (CDN)
+    keyman.addKeyboards('@th'); // Loads default Thai keyboard from Keyman Cloud (CDN)
```

Comment:
Why do we load EN and TH keyboards by default?

Reply:
This is an example only ... and long ago that was a good sample of keyboards. We could change it to another example one day.